rust/src/stracciatella.rs

```diff
@@ -544,6 +547,11 @@ pub fn should_start_in_fullscreen(ptr: *const EngineOptions) -> bool {
     unsafe_from_ptr!(ptr).start_in_fullscreen
 }
 
+#[no_mangle]
+pub fn should_use_integer_scaling(ptr: *const EngineOptions) -> bool {
+	unsafe_from_ptr!(ptr).use_integer_scaling
+}
+
```

Comment:
The indentation of new code doesn't match the rest of this file.

Reply:
What is the correct formating though? The readme says tab with 8 and to intend new code with tabs. But the current formatting suggests using spaces with a width of 4.

Reply:
Yes, tabs are fine, it's this file that's using spaces erroneously. But consistency is always better. There's ongoing work to harmonize everything to tabs (eg. #556 ).